roles/osg/tasks/main.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: Create osg role
+  os_keystone_role:
+    auth: "{{ openstack_auth }}"
+    name: osg
+    
+- name: "Add service user to osg project as osg"
+  os_user_role:
+    auth: "{{ openstack_auth }}"
+    user: "{{ osg_service_username }}"
+    role: osg
+    project: "{{ osg_project_id }}"
```

Comment:
Does it work to specify a project name/domain name here? That is probably easier to manage, as we only have to ensure the names are the same across sites for the play to be generic.